internal/cmds/iter.go

```diff
@@ -24,3 +27,10 @@ func (c XaddFieldValue) FieldValueIter(seq iter.Seq2[string, string]) XaddFieldV
 	}
 	return c
 }
+
+func (c ZaddScoreMember) ScoreMemberIter(seq iter.Seq2[float64, string]) ZaddScoreMember {
```

Comment:
```suggestion
func (c ZaddScoreMember) ScoreMemberIter(seq iter.Seq2[string, float64]) ZaddScoreMember {
```
Using [string, float64] will be much more convenient for users because the first value usually refers to a unique key. 

And Redis sorted set allows elements with the same score, so we better put float64 at the second value.